.eslintrc.json

```
@@ -13,6 +13,7 @@
   "globals": {
     "fastdom": true,
     "Promise": true,
-    "Set": true
+    "Set": true,
+	"Map": true
```

Mixing tabs and spaces
```suggestion
    "Set": true,
    "Map": true
```